Makefile.core.mk

```diff
@@ -33,12 +33,12 @@ export ISTIO_IMAGE_VERSION
 ISTIO_SHA ?= $(shell < ${ISTIOIO_GO}/go.mod grep 'istio.io/istio v' | cut -d'-' -f3)
 export ISTIO_SHA
 
-HUB ?= gcr.io/istio-testing
+HUB ?= gcr.io/istio-prerelease-testing
 ifeq ($(HUB),)
   $(error "HUB cannot be empty")
 endif
 
-TAG ?= $(shell echo "${ISTIO_IMAGE_VERSION}.${ISTIO_SHA}")
+TAG ?= 1.6.0-beta.0
```

Comment:
same with tag. What is `ISTIO_IMAGE_VERSION` used for? Is this the same thing as the container version?

Reply:
It was the old calculated name that never worked. The SHA was the short sha and they are named with the long sha.

Reply:
We now calculate the tag when run via the Makefile. There is an issue with the TAG coming in via the prow script so TAG is still hardcoded there.